common/defs/bitcoin/blockstamp.json

```diff
@@ -0,0 +1,44 @@
+{
+  "coin_name": "BlockStamp",
+  "coin_shortcut": "BST",
+  "coin_label": "BlockStamp",
+  "website": "https://blockstamp.info",
+  "github": "https://github.com/BlockStamp/bst",
+  "maintainer": "The BlockStamp Developers <support@blockstamp.info>",
```

Comment:
add your email; not some alias; you will be responsible for keeping the definition always up to date

Reply:
Thank you for comment. Already changed.